spec/services/trainees/create_from_apply_spec.rb

```diff
@@ -32,12 +35,7 @@ module Trainees
         diversity_disclosure: Diversities::DIVERSITY_DISCLOSURE_ENUMS[:diversity_disclosed],
         email: contact_details["email"],
         training_route: course.route,
-        course_subject_one: course.name,
```

Comment:
Again, just reminding you that I've decided to remove some attributes here because it will be copied over during the confirmation journey.